lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -397,6 +397,7 @@ class TestSampleValues(Commons):
     """
 
     SCROLL_ID = "random_scroll_id_string_1=="
+    RESULT_DATA_SAMPLE = "result-data-sample"
```

Comment:
Since this class extends `Commons`, you don't need to duplicate the symbol; and you can still reference it here as `self.RESULT_DATA_SAMPLE`.

Reply:
yeah. I thought the class is different.